src/packages/@library/ui/index.ts

```diff
@@ -9,3 +9,4 @@ export * from "./options";
 export * from "./label";
 export * from "./tooltip";
 export * from "./modal";
+export * from "./popover";
```

Comment:
add here a new export for Carousel also

currently you have pushed "carousel and popover" both in "popover" directory, create a separate directory for "carousel"